ext/reflection/php_reflection.c

```diff
@@ -1122,6 +1122,7 @@ static void reflection_attribute_factory(zval *object, HashTable *attributes, ze
 	reference->target = target;
 	intern->ptr = reference;
 	intern->ref_type = REF_TYPE_ATTRIBUTE;
+	ZVAL_STR_COPY(reflection_prop_name(object), data->name);
```

Comment:
why is this needed and not in https://github.com/php/php-src/commit/b0e83aabec6cdc30950a69068e82ec86fd68e564?

Reply:
Because there is already code that copies into the property table for the case you linked.

Reply:
My commit got reverted, I missed the fact that the property is inherited from `ReflectionFunctionAbstract`.